files/zh-tw/web/css/_colon_lang/index.md

```diff
@@ -1,26 +1,23 @@
 ---
-title: ':lang()'
+title: ":lang()"
 slug: Web/CSS/:lang
 ---
 
 {{CSSRef}}
 
 **`:lang()`** 是一種 [CSS](/zh-TW/docs/Web/CSS) [偽類 ( pseudo-class )](/zh-TW/docs/Web/CSS/Pseudo-classes)。它會根據括號中指定的語系，來選擇使用的元素。
 
-```css
-/* 選取任何語言屬性為英文 (en) 的 <p> */
-p:lang(en) {
-  quotes: '\201C' '\201D' '\2018' '\2019';
-}
-```
-
 > **備註：** 在 HTML 中，語言是通過 [`lang`](/zh-TW/docs/Web/HTML/Global_attributes#lang) 屬性和 {{HTMLElement("meta")}} 元素的組合來決定的, 也可能是通過協議的信息來確定 (例如 HTTP 表頭). 對於其他文檔類型，也可能存在其他用於確定語言的方法。
 
 ## 語法 Syntax
 
 ### 語法格式
 
-{{csssyntax}}
+```css-nolint
+:lang(<language-code>) {
+  /* ... */
+}
+```
```

Comment:
(Example of non-Prettier, content sync changes -- this one's a very small change that I feel confident to review, but there are others I'm not quite confident in reviewing!)

Reply:
Thanks for the note. I'll split the content sync changes to other PRs. I just forgot to list the pending review comments (sorry about this, but I still like to split them, not list those comments here)